services/src/main/resources/META-INF/services/org.keycloak.broker.provider.IdentityProviderMapper

```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 #
 
+org.keycloak.broker.provider.HardcodedGroupMapper
```

Comment:
It does not seem related but not hard on it :)

We really need to enable code style/format ...